tests/test_layering.py

```diff
@@ -21,13 +22,13 @@ def __init__(self):
                 self.add(LayeredSprite(x))
 
     scene = LayeredScene()
-    for lower_sprite, higher_sprite in zip(scene, list(scene)[1:]):
+    for lower_sprite, higher_sprite in zip(scene.sprite_layers(), list(scene.sprite_layers())[1:]):
```

Comment:
`itertools.islice()` might help here.

Reply:
Instead of the list > slice thing? Yeah, I can make that change.